examples/fluids/src/smartsim/sgs_dd_training.c

```diff
@@ -311,16 +318,20 @@ PetscErrorCode TSMonitor_SGS_DD_Training(TS ts, PetscInt step_num, PetscReal sol
     {
       const PetscScalar *training_data;
       PetscCall(VecGetArrayRead(TrainingData, &training_data));
+      PetscCall(PetscLogEventBegin(SmartRedis_Train, 0, 0, 0, 0));
       PetscSmartRedisCall(put_tensor(smartsim->client, array_key, array_key_len, (void *)training_data, sgs_dd_train->training_data_array_dims, 2,
-                                     SRTensorTypeDouble, SRMemLayoutContiguous));
+                                SRTensorTypeDouble, SRMemLayoutContiguous));
```

Comment:
I think this will fix some of the formatting issues.
```suggestion
                                     SRTensorTypeDouble, SRMemLayoutContiguous));
```